examples/tv-casting-app/android/App/app/src/main/jni/cpp/TvCastingApp-JNI.cpp

```diff
@@ -303,6 +303,20 @@ JNI_METHOD(jboolean, sendCommissioningRequest)(JNIEnv * env, jobject, jobject jD
     return true;
 }
 
+JNI_METHOD(jboolean, purgeCache)(JNIEnv * env, jobject)
+{
+    chip::DeviceLayer::StackLock lock;
+    ChipLogProgress(AppServer, "JNI_METHOD purgeCache called");
+
+    CHIP_ERROR err = CastingServer::GetInstance()->PurgeCache();
```

Comment:
on iOS this API is async, but on Android its sync - any reason for that?

Should we make this async too or alternatively make the other one sync?

The APIs should only differ if there's a good reason for it

Reply:
Made the iOS API sync (did not need to be async, because this is a quick operation on the disk on iOS.